container/dockershell.sh

```diff
@@ -145,6 +150,11 @@ if [ -z ${DEPLOYMENT_TYPE} ]; then
     DEPLOYMENT_TYPE=$defaultDeployType
 fi
 
+if [ -z ${ES_VERSION} ]; then
+    printf "Event Store version (--es-version) not specified, please provide"
```

Comment:
I would indicate that we will use the default version (even mentioning that version explicitly), rather than saying "please provide"

Reply:
I am going to remove this as in the setup-container script i test if a version was specified and if not it uses 2.0.1.0. Which means this script will need to be updated when the json file is update with the next realese. 

Perhaps the docs should be explicit to which version is current and that is what the scripts will use?

Reply:
Sounds good, update the README though to indicate what happens when it's not specified.